src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -17,6 +17,7 @@ tags:
   - name: Person
   - name: Published Binaries
   - name: Requests
+  - name: Sources
```

Comment:
It was removed in [this commit](https://github.com/openSUSE/open-build-service/commit/81f3fdff94c7fbc2a9d2700122dd9c18059f03c5#diff-941da4259ba92323e54fa18add5045aeed9001f3e26c9e7b4040e9c335f25331). Probably because of conflicts. We need to keep it. :man_facepalming: